compiler/damlc/BUILD.bazel

```diff
@@ -79,6 +79,7 @@ package_app(
     name = "damlc-dist",
     binary = ":damlc",
     resources = [
+        ":daml-base-anchors.json",
```

Comment:
I'm surprised this works! It looks like `daml-base-anchors.json` is created by the `damlc` binary, so `damlc-dist` can indeed take artifacts produced by `damlc`. Neat!